recipes/sarpy/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
The PyPI tarball used here already includes `LICENSE`, so you shouldn't add this manually. Please remove.